apps/service_persist/test/persist_test.exs

```diff
@@ -159,12 +147,8 @@ defmodule PersistTest do
         id: "persist-1",
         dataset_id: "ds1",
         subset_id: "example",
-        source: "topic-example",
-        destination: "ds1_example",
-        schema: [
-          %Dictionary.Type.String{name: "name"},
-          %Dictionary.Type.Integer{name: "age"}
-        ]
+        source: Source.Fake.new(),
+        destination: "ds1_example"
```

Comment:
Don't both of these need to be the new style of protocol struct?

Reply:
Not yet. Kafka is our only destination so far. This is a presto destination.